Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -141,7 +141,7 @@ import Foundation
             case storeAppStore = "app_store"
             case storeMacAppStore = "mac_app_store"
             case storePlayStore = "google_play_store"
-            case testStore = "test_store"
+            case simulatedStore = "simulated_store"
```

Comment:
If I'm not mistaken, this has not been added in Android yet.

I don't know if it's useful to add this now as I don't think we're getting this information for Customer Center yet, but it can be good for future proofing I guess. Also, I think "simulated_store" would be the decided String for this. We could change this at any time if needed. WDYT @RevenueCat/coresdk?